src/librustdoc/doctest.rs

```diff
@@ -249,14 +265,82 @@ pub(crate) fn run(
 pub(crate) fn run_tests(
     mut test_args: Vec<String>,
     nocapture: bool,
-    mut tests: Vec<test::TestDescAndFn>,
+    opts: GlobalTestOptions,
+    rustdoc_options: &Arc<RustdocOptions>,
+    unused_extern_reports: &Arc<Mutex<Vec<UnusedExterns>>>,
```

Comment:
nit: Use `clone()` instead of `&` since these are `Arc`s.

Reply:
`clone` has a (small) cost so in this case, it's not impacting the code badly and in some cases, they are not cloned so that's always one less (small) `clone`... (weird to name that clone and not `increase_clones_counter` but whatever)